lib/active_model/serializer/fieldset.rb

```diff
@@ -21,7 +21,7 @@ def fields_for(type)
 
       def parsed_fields
         if raw_fields.is_a?(Hash)
-          raw_fields.inject({}) { |h, (k, v)| h[k.to_sym] = v.map(&:to_sym); h }
+          raw_fields.each_with_object({}) { |(k, v), h| h[k.to_sym] = v.map(&:to_sym) }
```

Comment:
This (isomorphic) change is not necessary for this PR, but it makes the code slightly cleaner.